addons/account/tests/test_account_move_entry.py

```diff
@@ -269,23 +269,26 @@ def test_misc_draft_reconciled_entries_1(self):
 
         (lines[0] + lines[2]).reconcile()
 
-        draft_moves.flush()
-        self.cr.execute('SAVEPOINT test_misc_draft_reconciled_entries_1')
-
-        with self.assertRaises(UserError):
+        # You can't write something impacting the reconciliation on an already reconciled line.
+        with self.assertRaises(UserError), self.cr.savepoint():
             draft_moves[0].write({
                 'line_ids': [
                     (1, lines[1].id, {'credit': lines[1].credit + 100.0}),
                     (1, lines[2].id, {'debit': lines[2].debit + 100.0}),
                 ]
             })
 
-        with self.assertRaises(UserError):
-            draft_moves.unlink()
+        # Make sure the monetary fields are well managed when checking the reconciliation.
```

Comment:
Be more explicit about what 'well managed' means. ("What do you mean by 'well managed' ?" :p ) You could as well just say you expect the write not to raise anything because the rounding will ignore such a tiny amount